src/main/resources/fileTemplates/common_build.gradle.kts.ft

```diff
@@ -56,6 +59,14 @@ kotlin {
                 #else
                 api(compose.material)
                 #end
+
+                #if (${USE_KTOR})
+                implementation("io.ktor:ktor-client-core:$ktorVersion")
+                #end
+
+                #if (${USE_KOIN})
+                implementation("io.insert-koin:koin-core:$koinVersion")
```

Comment:
If Koin doesn't work with the web configuration, we need to:
- handle that
- not use Koin
- only allow enabling Koin when web is disabled
With these options, I'd prefer not to include Koin since this plugin should set up all the configurations with all the dependencies.

Reply:
Just wanted to bring it up just in case 👍

Reply:
As long as it works I'm good.